man/systemd.special.xml

```diff
@@ -503,8 +514,11 @@
       <varlistentry>
         <term><filename>sysinit.target</filename></term>
         <listitem>
-          <para>A special target unit covering early boot-up
-          scripts.</para>
+          <para>This target pulls in the services required for system
+          initialization. System services pulled in by this target should
+          be prepared to work without all mount points, and should declare
```

Comment:
I am currently thinking "needs anything more than a read-only root", that is probably more understandable.